test/playwright.spec.js

```diff
@@ -180,12 +180,12 @@ module.exports.addPlaywrightTests = ({testRunner, platform, products, playwright
           state._stdout.on('line', dumpout);
           state._stderr.on('line', dumperr);
           if (dumpProtocolOnFailure)
-            state.browser._setDebugFunction(data => test.output.push(`\x1b[32m[pw:protocol]\x1b[0m ${data}`));
+            debug('pw:protocol').output = data => test.output.push(`\x1b[32m[pw:protocol]\x1b[0m ${data}`);
           state.tearDown = async () => {
             state._stdout.off('line', dumpout);
             state._stderr.off('line', dumperr);
             if (dumpProtocolOnFailure)
-              state.browser._setDebugFunction(() => void 0);
+              debug('pw:protocol').output = data => console.log;
```

Comment:
Not sure what this meant to be, but looks suspicious :)

Reply:
Fixed